### PR TITLE
[api-minor] Slightly shorten some `idFactory` strings

### PR DESCRIPTION
**Pull request by Snuffleupagus**

### CODE REVIEW

File: src/core/font_substitutions.js [hunk 1]
```diff
@@ -400,8 +400,6 @@ function getFontSubstitution(
   baseFontName,
   standardFontName
 ) {
-  let mustAddBaseFont = false;
-
```
Comment (Snuffleupagus) [inline src/core/font_substitutions.js:404]: *Please note:* Moving this down is unrelated to the rest of the patch, however I happened to notice that initializing this variable is unnecessary for already *cached* standard-fonts.

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Looks good; thanks!
